tutorial.md

```diff
@@ -241,30 +294,45 @@ class MyButton extends StatelessComponent {
 }
 ```
 
-The `GestureDetector` widget doesn't have an visual representation but instead
-detects gestures made by the user. When the user taps the `Container`, the
-`GestureDetector` will call its `onTap` callback, in this case printing a
-message to the console. You can use `GestureDetector` to detect a variety of
-input gestures, including taps, drags, and scales.
-
-Many components use a `GestureDetector` to provide optional callbacks for other
-components. For example, the `IconButton`, `RaisedButton`, and
-`FloatingActionButton` components have `onPressed` callbacks that are triggered
-when the user taps the widget.
+The
+[`GestureDetector`](https://domokit.github.io/docs/sky/material/GestureDetector-class.html)
+widget doesn't have an visual representation but instead detects gestures made
+by the user. When the user taps the
+[`Container`](https://domokit.github.io/docs/sky/material/Container-class.html),
+the
+[`GestureDetector`](https://domokit.github.io/docs/sky/material/GestureDetector-class.html)
+will call its
+[`onTap`](https://domokit.github.io/docs/sky/material/GestureDetector-class.html#onTap)
+callback, in this case printing a message to the console. You can use
+[`GestureDetector`](https://domokit.github.io/docs/sky/material/GestureDetector-class.html)
+to detect a variety of input gestures, including taps, drags, and scales.
+
+Many components use a
+[`GestureDetector`](https://domokit.github.io/docs/sky/material/GestureDetector-class.html)
+to provide optional callbacks for other components. For example, the
+[`IconButton`](https://domokit.github.io/docs/sky/material/IconButton-class.html),
+[`RaisedButton`](https://domokit.github.io/docs/sky/material/RaisedButton-class.html),
+and
+[`FloatingActionButton`](https://domokit.github.io/docs/sky/material/FloatingActionButton-class.html)
+components have
+[`onPressed`](https://domokit.github.io/docs/sky/material/RaisedButton-class.html#onPressed)
+callbacks that are triggered when the user taps the widget.
 
 Managing State
 --------------
 
 Thus far, we've used only stateless components. Stateless components receive
-arguments from their parent component, which they store in `final` member
-variables. When a component is asked to `build`, it uses these stored values to
-derive new arguments for the widgets it creates.
+arguments from their parent component, which they store in
+[`final`](https://www.dartlang.org/docs/dart-up-and-running/ch02.html#final-and-const)
+member variables. When a component is asked to
+[`build`](https://domokit.github.io/docs/sky/material/StatelessComponent/build.html),
+it uses these stored values to derive new arguments for the widgets it creates.
 
 In order to build more complex experiences, however - for example, to react
```

Comment:
@krisgiesing - can we drop the "however" here?  ", however - for example -" is awkward to read.